robotbase/src/main/java/frc/robot/Robot.java

```diff
@@ -19,14 +28,46 @@ public class Robot extends TimedRobot {
 
   private final RobotContainer m_robotContainer;
 
+  public static CommandSwerveDrivetrain swerve;
+  public static AutoChooser autoChooser;
+
+  private Autos m_autos;
+
+  private Map<String, Command> m_autoCommandsToBind;
+  private Map<String, AutoRoutine> m_autoRoutinesToBind;
+
   /**
    * This function is run when the robot is first started up and should be used for any
    * initialization code.
    */
   public Robot() {
-    // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
-    // autonomous chooser on the dashboard.
     m_robotContainer = new RobotContainer();
+
+    swerve = TunerConstants.createDrivetrain();
+    autoChooser = new AutoChooser();
+
+    m_autos = new Autos();
+
+    SendableBuilderImpl autoChooserBuilder = new SendableBuilderImpl();
```

Comment:
Let's wrap all this auto chooser work in a class. Robot.java should be for high-level config only. Instantiate the class here, call a method to get the auto command in autonomousInit()